### DISCUSSION TITLE
Bitcoin Miner MEV

### DISCUSSION THREAD

**Discussion by igorsyl** [stacks-network/stacks-core#3619] [General]
Starting a discussion about growing incidence of Bitcoin Miner MEV

Initial discussion: https://discord.com/channels/621759717756370964/728030782693834862/1085274251944284290

**Comment by badonyx** [comment]
One of the top 4 Bitcoin mining pools is removing block commit transactions from other STX miners and submitting their own when they win BTC blocks, in order to win easy STX block rewards without any significant PoX commitment. This only works some of the time however since the other miners mostly continue building off of the old tip. 

<img width="1173" alt="Screenshot 2023-03-16" src="https://user-images.githubusercontent.com/110633279/225788857-481bec1b-8405-4421-a8f3-10cb8479af42.png">

This results in inefficiencies in either scenario:
- Transient consensus failures, where the MEV block is forked away, denying their block reward but also resulting in a reorg and denying Stacks consensus from settling on that Bitcoin block.
- Successful MEV but other miners waste their block commitments on a dead fork in a subsequent block.

**Reply by badonyx** [reply]
At time of writing the MEV miner has made 775 block commit transactions, 591 of which were in blocks including only their own PoX transaction, while winning only 166 block rewards. The remainder indicates transient consensus failure at rate of at least 72% per MEV block (which they win between 10-15% of the time).

**Comment by friedger** [comment]
The current MEV miner does not care about tx fees and does not include any txs.

**Reply by badonyx** [reply]
This is not true.

<img width="1006" alt="Screenshot 2023-03-17 2" src="https://user-images.githubusercontent.com/110633279/225982030-9e2474bb-029c-464c-811d-2c906f93d913.png">

**Reply by friedger** [reply]
Good to see!

**Comment by igorsyl** [comment]
This topic was discussed in today's (2023-03-17) Blockchain Architecture Meeting. [Meeting Notes](https://docs.google.com/document/d/1_glu_Xu9KkJejMsGtpSwJ_xqOuO8SmHN1hCo7VYdJmQ/edit).

**Comment by jcnelson** [comment]
Let's get this ball rolling: https://forum.stacks.org/t/rfc-minimum-bid-pox-to-thwart-bitcoin-mev/14675
